File: android/supplier/DeliveryApp/app/src/main/java/hu/bme/aut/android/deliveryapp/model/JobDetails.kt
package hu.bme.aut.android.deliveryapp.model

import java.io.Serializable

data class JobDetails(
    val title: String?,
    val description: String?,
    var clientId: String, //TODO: should be val
    var transporterId: String?,
    var deliveryId: String, //TODO: should be val
    val clientName: String?,
    val clientAvgRating: Float?,
    val imPaths: String?,
    val deliveryDate: String?,
    val deliveryCost: Int?,
    val deliveryLocation: Location?,
) : Serializable

File: backend/src/types/JobDetails.ts
import { ILocation } from './location'

export interface IJobDetails {
  title: string
  description: string
  clientId: string
  transporterId?: string
  deliveryId: string
  clientName: string
  clientAvgRating: number
  imPaths: string
  deliveryDate: Date
  deliveryCost: number
  deliveryLocation: ILocation
}
